<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="framefondo">
      <property name="styleSheet">
       <string notr="true">QWidget#framefondo{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.039548 rgba(0, 67, 109, 255), stop:1 rgba(203, 203, 203, 255));}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_superior">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>911</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>331</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Arial&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Gestión de inquilinos</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnRegistrar">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>380</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../Downloads/image-removebg-preview (4).png</normaloff>../../../../../Downloads/image-removebg-preview (4).png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Ingrese su cédula:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCedula">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>130</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNombre">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>200</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Apellido 1:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>200</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Apellido 2:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtApellido1">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>220</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtApellido2">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>220</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>280</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Télefono:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTelefono">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>300</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>280</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Correo:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCorreo">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>300</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Registrar nuevo usuario</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
